### DISCUSSION TITLE
Vertical tabs with custom content align to left

### DISCUSSION THREAD

**Discussion by robert-niestroj** [quasarframework/quasar#14129] [General - Components / Directives / etc] [answered]
Hi, i have want to use vertical tabs https://v1.quasar.dev/vue-components/tabs#vertical with custom tab content. 
In this custom tab content i want some text aligned to the left, but an icon aligned to the right like this: 

![image](https://user-images.githubusercontent.com/9776384/182565563-cc40e87c-50e9-4c39-a8b0-e29ff33d2e93.png)

I use the provided slot, but my row does not expand to the full width. 
Aligning left was already a challenge. I had to use some 'strange' styling: `justify-content:initial`
I also tried `full-width` but no success. 

Here's what i tried:
https://codepen.io/robert-niestroj/pen/jOzZoVb?editors=101

How do i get the row to expand to full width?

**Comment by pdanpdan** [comment, marked as answer]
Check the changes for first tab
https://codepen.io/pdanpdan/pen/WNzMVpa?editors=1000

**Reply by robert-niestroj** [reply]
Big "Thank you!"
